MatrixMultiplication/MatrixMultiplication/MatrixMultiplier.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2024
+//
+// Use of this source code is governed by an MIT license
+// that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+namespace MatrixMultiplication;
+
+/// <summary>
+/// Povides methods for matrix multiplication.
+/// </summary>
+public static class MatrixMultiplier
+{
+    /// <summary>
+    /// Reads matrix from file into two-dimensional array.
+    /// </summary>
+    /// <param name="filepath">filepath to matrix.</param>
+    /// <returns>two-dimensional array representing the given matrix.</returns>
+    public static int[,] InputMatrix(string filepath)
+    {
+        var streamReader = new StreamReader(filepath);
```

Comment:
```suggestion
        using var streamReader = new StreamReader(filepath);
```
Чтобы ресурс не блокировать